Localization/Resources/sv-SE/winget.resw

```diff
@@ -628,7 +628,7 @@ De kan konfigureras via inställnings filen "winget settings".</value>
     <value>Okänt kommando</value>
   </data>
   <data name="Usage" xml:space="preserve">
-    <value>användning</value>
+    <value>Användning</value>
```

Comment:
Should this be capitalized?  You can see when it is used with:
```
> winget -?
Windows Package Manager v0.2.2941 Preview
Copyright (c) Microsoft Corporation. All rights reserved.

The winget command line utility enables installing applications and other packages from the command line.

usage: winget [<command>] [<options>] <- !!Beginning of this line!!
...
```

I don't know if Swedish would expect this to be capitalized in this case (my grandmother never taught me any, as she was only 2 when she came to the US :smile:).

Reply:
You're absolutely correct. I think I just presumed that it was a miss and didn't consider the context. I pushed up a fix for it now. 

Fun that you have Swedish heritage, have you ever visited?

Reply:
No, though I've often thought it would be an interesting place to go (in the summer).  My mother and aunt both went and found the likely location where my great grandparents lived, although records are sparse and I believe the only thing they had to confirm was older locals claiming that someone of that family name living there long ago.